### ISSUE TITLE
 AttributeError: 'NoneType' object has no attribute 'python_grad_func'

### ISSUE THREAD

**Issue by nikunj95** [tensorflow/hub#668]
Hello,

I am trying to use Trill module by integrating it with the Keras Sequential model but I am facing an issue and I am not able to resolve it.

![Screen Shot 2020-09-08 at 4 27 23 PM](https://user-images.githubusercontent.com/26219520/92524552-4fb61700-f1f0-11ea-85c9-bf590831cff9.png)

This is the error that I am facing.

![Screen Shot 2020-09-08 at 4 29 26 PM](https://user-images.githubusercontent.com/26219520/92524667-796f3e00-f1f0-11ea-8c65-db3389d626d4.png)

This is the simple code that I wrote to test the integration of the module with a layer of LSTM and Dense layer. The Trill hub module gives output of [ batch size, time, features ]. 

I would appreacite if you could help me out with this issue or point me to the right sources so that I can understand the error and resolve it.
Thank you!

**Comment by rmothukuru** [comment]
@nikunj95,
The code where you load Trill Module is missing. Can you please paste the code instead of Screenshot so that it would be easy for us to copy and reproduce it at our end? Thanks!

**Comment by nikunj95** [comment]
Thank you for your reply. Here's the sample code which will help you to reproduce the issue.
`import tensorflow as tf
import tensorflow_hub as hub
import numpy as np

# Below code is just to reproduce the error - my X are output from wav file and y are label
X = tf.convert_to_tensor( np.sin(np.linspace(-np.pi, np.pi, 128), dtype=np.float32) )
X = tf.reshape( X, (1, 128) )
y = tf.convert_to_tensor( [1] )
y = tf.reshape( y, (1, 1) )

# Load the Trill module into the Keras Layer
module = hub.load( 'https://tfhub.dev/google/nonsemantic-speech-benchmark/trill/3' )
hub_layer = hub.KerasLayer( module, arguments={ 'sample_rate': 16000 }, output_key='embedding', trainable=False )

# Model
model = tf.keras.Sequential()
model.add( hub_layer )
model.add( tf.keras.layers.LSTM(100) )
model.add( tf.keras.layers.Dense( 8, activation='softmax' ) )

model.compile(
    optimizer='adam',
    loss='sparse_categorical_crossentropy',
    metrics=['accuracy']
)
model.fit( X, y, epochs=3, verbose=1 )

# ERROR-> AttributeError: 'NoneType' object has no attribute 'python_grad_func'`

Hope you can reproduce the error and help me resolve it.

**Comment by andresusanopinto** [comment]
Thanks for the clean repro @nikunj95!

TLDR: It is fixed in tf-nightly pip package and should eventually be in 2.4 when it is released.

Notes:
 - I verified it reproduces in the latest stable tensorflow release (2.3.1).
 - I verified it runs in the latest tf-nightly ("2.4.0-dev20201001").
